canonical-web-and-design/ubuntu.com
# Improved data insertion and fixed relationships

## Done

- An improved version of the data migration PR. If this works, we should close #7100 
- All data I could add has been added to the first CVE-2020

## QA

- Check out this feature branch
- Run the site with dotrun snap
- `docker-compose up`
- Visit page `/security/cve`
- Repopulate data with `dotrun exec python3 webapp/security/fixtures/load_sample_cve.py`

## Issue / Card

Fixes #7030
